config/opensearch_dashboards.yml

```diff
@@ -253,6 +253,9 @@
 #   faviconUrl: ""
 #   applicationTitle: ""
 #   useExpandedHeader: false
+#   AILogo:
+#     grayUrl: ''
+#     gradientUrl: ''
```

Comment:
Can we move this config to `dashboards-assistant`? we already have `assistant.branding.logo`, we can update from there.
Since we want AI related features and configuration be in `dashboards-assistant`, and we will move query assistant feature to `dashboards-assistant` plugin. So I'll be very caution to add assistant related config in OSD core.